src/containers/ExploreContainer.jsx

```diff
@@ -1,22 +1,54 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import {
   View,
   ScrollView,
 } from 'react-native';
 
+import { useQuery } from '@apollo/react-hooks';
 import SearchBar from '../components/SearchBar';
 import FriendTile from '../components/FriendTile';
+import {
+  FRIEND_SEARCH,
+} from '../graphql/GeneralQueries';
 
 export default function ExploreContainer() {
+  const [searchText, setSearchText] = useState('');
+  const [friendTiles, setFriendTiles] = useState(<View />);
```

Comment:
Okay for the time being. Ideally I think you would create a data structure to manage friends and in the return method just map that friend data structure to friend tiles. As opposed to managing a state variable which itself is JSX